spec/support/methods/feature_test_helper_methods.rb

```diff
@@ -0,0 +1,45 @@
+module FeatureTestHelperMethods
+  def start(the_flow:, at:)
+    visit "/#{at}"
+    ensure_page_has(header: the_flow)
+    click_govuk_start_button
+  end
+
+  def answer(question:, of_type:, with:)
+    ensure_page_has(header: question)
+    case of_type
+    when :radio_button
+      choose with
+    when :check_box
+      check with
+    when :check_boxes
+      raise "For the `check_boxes` type, `with` should be an array" unless with.is_a?(Array)
+
+      with.each do |answer|
+        check answer
+      end
+    when :input_text
+      fill_in "response", with: with
+    when :date
+      raise "For the `date` type, `with` should be a hash" unless with.is_a?(Hash)
+
+      fill_in "response[day]", with: with[:day]
+      fill_in "response[month]", with: with[:month]
+      fill_in "response[year]", with: with[:year]
+    end
```

Comment:
Might be helpful to have a catch all (`else`) clause on the switch that raises an error in case we input an invalid `type_of` by mistake. It was somewhat difficult to realise when I'd done this, because the errors in capybara are pretty confusing.

Reply:
Yeah, I agree - will add this in.